Automattic/themes
# Blank Canvas: Add additional "Link in Bio" block patterns

Partially addresses #3161.

This PR adds a new "Link in Bio" category of block patterns, and populates it with all of the un-blocked patterns from #3161:

- Emoji Links
- Gradient Links
- Musician Links
- Product Links
- Text Links

---

To test: 

1. Open the Block Patterns Panel.
2. Select the "Link in Bio" category.
3. Insert the included patterns. 
4. Check for any errors on the front and backend. 

---

## Screenshots

![patterns](https://user-images.githubusercontent.com/1202812/106659832-3fbb1d00-656d-11eb-969a-385d2a1d582b.png)
<img width="1133" alt="Screen Shot 2021-02-02 at 3 40 42 PM" src="https://user-images.githubusercontent.com/1202812/106659837-421d7700-656d-11eb-9886-6918fc340cd2.png">
![patterns-2](https://user-images.githubusercontent.com/1202812/106659842-434ea400-656d-11eb-89fd-a9b3be90edbb.png)
